tests/wrappers/django_test/stress.py

```diff
@@ -0,0 +1,30 @@
+import time
+import requests
+import threading
+
+ANSWER_A = b'This is A'
+ANSWER_B = b'This is B'
+
+def threadA(name):
+    time.sleep(2)
+    response = requests.get("http://127.0.0.1:8000/polls/a")
+    if response.content != ANSWER_A:
+    	print("Bad answer in A!")
+    	print(response.content)
+
+def threadB(name):
+    time.sleep(2)
+    response = requests.get("http://127.0.0.1:8000/polls/b")
+    if response.content != ANSWER_B:
+    	print("Bad answer in B!")
+    	print(response.content)
+
+def main():
+    a = threading.Thread(target=threadA, args=(1,))
```

Comment:
Oh, this one is the stress test 👍 
How about a loop repeating it in main?
Also, does the other (wrapper) test have any added value?

Reply:
Note that this is a manual test as we wanted to release this fix ASAP. We are currently working on adding a similar test that will run on each PR as a sanity/regression test.